src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -448,10 +494,34 @@ def build(self, notification_uuid: str | None = None) -> Union[SlackBlock, Slack
             action_block = {"type": "actions", "elements": [action for action in actions]}
             blocks.append(action_block)
 
+        # add mentions
+        if self.mentions:
+            mentions_text = f"Mentions: {self.mentions}"
+            blocks.append(self.get_markdown_block(mentions_text))
```

Comment:
i guess this is changing to "description" now based on dhrumil's comments?

Reply:
I'm gonna change this in a separate PR since I need to touch the alert config files too